internal/slicer/report.go

```diff
@@ -11,12 +11,14 @@ import (
 )
 
 type ReportEntry struct {
-	Path   string
-	Mode   fs.FileMode
-	Hash   string
-	Size   int
-	Slices map[*setup.Slice]bool
-	Link   string
+	Path      string
+	Mode      fs.FileMode
+	Hash      string
+	Size      int
+	Slices    map[*setup.Slice]bool
+	Link      string
+	Mutated   bool
```

Comment:
Was that in the spec?   Do we need "mutated" or "mutable"?  One of them says it's okay for the content to change, while the other says the content has changed.  The latter is just final != initial, while the former we cannot compute without data.

Reply:
This is not in the spec and it will not be persisted in the db, it is only there to know that a file has changed. Because the `final_sha256` is defined as:
> the checksum of the file after it has been modified during installation (in hex format). This attribute is required only for files that have been mutated.

We need the flag to know if a file had been mutated. We cannot rely on final != initial because the content could be the same, but is was mutated.

Reply:
I guess it depends on the definition of mutated that we are using. You are saying mutated means content is not the same as the original whereas I am saying mutated is when a script writes to the file. For the latter, it could happen that the script writes the same content than what was in the file.

I think recording mutation regardless of content is slightly more robust because the db will reflect that the file was written to in all occasions and not based on the content.

Reply:
Agreed to not use `mutated` and to only report final hash when final hash != hash.